### PR TITLE
add: delete specialization doctor type

### LINKED ISSUE #2832: Doctor List in Facility page : Unable to delete
once, we add an doctor's area of specialization in a facility, we have no option to delete the specialization from the page

### PR DESCRIPTION
**Pull request by kunatastic**
closes #2832 

https://user-images.githubusercontent.com/61267953/176882626-3ab938d1-bb79-4d41-bda2-f02dcbef2cc2.mp4

Added the delete button directly on facilty page because
1. The edit form was being re-used at several places
2. Now, both the edit and delete option are at same location

### CODE REVIEW

**Review by rithviknishad — CHANGES_REQUESTED** [review]
@kunatastic 

When I delete all items, it does not show "No Doctors Found".
image

However, when I refresh the page only it shows up.
image

**Comment by kunatastic** [comment]
Cool @rithviknishad, on it

**Review by cp-Coder — APPROVED** [review]

**Comment by nihal467** [comment]
@kunatastic now the UI look so lengthy, is there any alternative way to implement the delete button

**Review by rithviknishad — APPROVED** [review]

**Comment by kunatastic** [comment]
Updated UI
image

image

@nihal467 is this better?

**Comment by nihal467** [comment]
test approved

**Review by khavinshankar — APPROVED** [review]

File: src/Components/Facility/models.tsx [hunk 1]
```diff
@@ -47,6 +47,7 @@ export interface FacilityModel {
   state_object?: StateModel;
   ward_object?: WardModel;
   modified_date?: string;
+  created_date?: string;
```
Comment (khavinshankar) [inline src/Components/Facility/models.tsx:50]: @kunatastic should this be outside the bracket

File: src/Components/Facility/models.tsx [hunk 2]
```diff
@@ -47,6 +47,7 @@ export interface FacilityModel {
...
 }
 
 export interface CapacityModal {
```
Comment (khavinshankar) [inline src/Components/Facility/models.tsx:53]: check this too (repetitions)

**Review by khavinshankar — CHANGES_REQUESTED** [review]

**Review by khavinshankar — APPROVED** [review]

**Pull request merged**